scio-jdbc/src/main/scala/com/spotify/scio/jdbc/JdbcIO.scala

```diff
@@ -104,38 +144,42 @@ final case class JdbcSelect[T: Coder](readOptions: JdbcReadOptions[T]) extends J
     EmptyTap
 }
 
-final case class JdbcWrite[T](writeOptions: JdbcWriteOptions[T]) extends JdbcIO[T] {
+final case class JdbcWrite[T](opts: JdbcConnectionOptions, statement: String) extends JdbcIO[T] {
```

Comment:
We can think of merging the read/write into a single IO that now they have the same signature. Probably for a next step

Reply:
That would be a good thing